apps/nextjs-website/src/lib/fetch/types/PageSection.ts

```diff
@@ -262,6 +262,22 @@ const EditorialSwitchSectionCodec = t.strict({
   reversed: t.boolean,
 });
 
+const VideoSectionCodec = t.strict({
+  __component: t.literal('sections.video'),
+  title: t.string,
+  subtitle: t.string,
+  src: StrapiImageRequiredSchema,
```

Comment:
Not a change, but a note.

We may change the type of this to make it clearer that it can be both a video and a image.